biz.aQute.bndlib/src/aQute/bnd/osgi/resource/CapReqBuilder.java

```diff
@@ -38,6 +41,10 @@
 
 public class CapReqBuilder {
 
+	private static final String			REQ_ALIAS_BUNDLE				= "bundle";
```

Comment:
`bundle` seems too short. We should at least scope with `bnd.bundle`. This also makes it clear this is a bnd-ism.